server/worker.go

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-vela/types"
+	"github.com/go-vela/types/library"
+)
+
+const (
+	// WorkerResp represents a JSON return for a single worker.
+	WorkerResp = `
+		{
+			"id": 1,
+			"hostname": "worker_1",
+			"address": "http://vela:8080",
+			"routes": [
+			"large",
+			"docker",
+			"large:docker"
+			],
+			"active": true,
+			"last_checked_in": 1602612590
+		}`
+
+	// WorkersResp represents a JSON return for one to many workers.
+	WorkersResp = `[
+		{
+			"id": 1,
+			"hostname": "worker_1",
+			"address": "http://vela:8080",
+			"routes": [
+			  "large",
+			  "docker",
+			  "large:docker"
+			],
+			"active": true,
+			"last_checked_in": 1602612590
+		  },
+		{
+			"id": 2,
+			"hostname": "worker_2",
+			"address": "http://vela:8081",
+			"routes": [
```

Comment:
Might make sense to show these as a different dataset than the ones above

Reply:
Can you clarify which you think should be in a separate dataset?

I'm not sure if you mean the routes or something else.